.gitmodules

```diff
@@ -2,7 +2,7 @@
 	path = 3rdparty/HighFive
 	url = https://github.com/BlueBrain/HighFive.git
 [submodule "3rdparty/pybind11"]
-	path = 3rdparty/pybind11
+	path = binds/python/pybind11
```

Comment:
why the move here?

Reply:
The way it is arranged currently, an if clause is needed in the `3rdparty/` to optionally add the `pybind11` subdirectory.

By moving it to the python dir, all the Python-related stuff is in the same place and only the root level inclusion of the subdirectory is needed in the case of building the wheels. It also reflects how libsonata is organized wrt the python bindings.

Reply:
I see, I think the `MANIFEST.in` probably needs to be updated then.

When making a change like this, it's worth checking the the sdist builds fine.

Reply:
Good catch. I updated the `MANIFEST.in` and tested the sdist build.